jobs/service-fabrik-broker/spec

```diff
@@ -197,6 +199,12 @@ properties:
   broker_drain_message:
     description: "Drain message that is updated into broker maintenance state. If undefined, then during drain maintenance state is not updated"
     default: "BROKER_DRAIN_INITIATED"
+  allow_concurrent_operations:
+    description: "When set to true, it will disable the check for parallel create/update/delete operation during any OSB operation"
```

Comment:
```suggestion
    description: "When set to true, it will disable the check for parallel create/update/delete operation of the service instance"
```
I felt a bit confusing.